sabnzbd/utils/correct_extension.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/python3
+
+""" function to check and find correct extension of a (deobfuscated) file
+"""
+
+
+import puremagic
+import os
+
+# common extension from https://www.computerhope.com/issues/ch001789.htm
+commonextlist = [
```

Comment:
For consistency with the rest of SAB's code-base, we capitalize constants: `COMMON_EXT` and `DOWNLOAD_EXT`.
And let's also use a tuple instead of a list, since they are constants (so `( )` instead of `[ ]`).

Reply:
Thanks. Please review my new implementation.